src/client/helpers/utils.tsx

```diff
@@ -171,6 +171,34 @@ export function dateObjectToISOString(value: DateObject) {
 	return date;
 }
 
+/**
+ * Remove trailing parantheses from a url string.
+ * @function removeTrailingParantheses
+ * @param {string} url - URL string.
+ * @returns {string} cleanUrl - sans any trailing parantheses.
+ */
+function removeTrailingParantheses(url: string, count: number): string {
+    return url.slice(0, -1*count);
```

Comment:
How would this deal with something like `https://example.com/wiki/This_is_a_test_(Test))#2`. If I’m reading the code correctly, it would produce `https://example.com/wiki/This_is_a_test_(Test))#` instead of the desired `https://example.com/wiki/This_is_a_test_(Test)`.